samples/openthread/ncp/prj.conf

```diff
@@ -0,0 +1,143 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+# ===============================
+# KERNEL OPTIONS
+# ===============================
+CONFIG_NEWLIB_LIBC=y
+CONFIG_MAIN_STACK_SIZE=4096
```

Comment:
I'd try to finetune this and keep it as low as possible - all main thread is doing is an app banner printout. We might need some extra for initialization though.

Reply:
Done